instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/route/RouteConcatenationInstrumentation.java

```diff
@@ -33,11 +33,11 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class ApplyAdvice {
 
+    @Advice.AssignReturned.ToArguments(@Advice.AssignReturned.ToArguments.ToArgument(0))
```

Comment:
@SylvainJuge usually writes this as `@AssignReturned.ToArguments(@ToArgument(0))`